src/rpc/util.cpp

```diff
@@ -575,29 +575,25 @@ std::string RPCArg::ToDescriptionString() const
             ret += "json array";
             break;
         }
-
-            // no default case, so the compiler can warn about missing cases
-        }
+        } // no default case, so the compiler can warn about missing cases
     }
     if (m_fallback.which() == 1) {
         ret += ", optional, default=" + boost::get<std::string>(m_fallback);
     } else {
         switch (boost::get<RPCArg::Optional>(m_fallback)) {
         case RPCArg::Optional::OMITTED: {
```

Comment:
In commit "refactor: Replace confusing OMITTED_NAMED_ARG with OMITTED" (fab2f3bb6446d5202b7a665cde1e7c0d02718e18)

For a future followup I think it'd be good to consider renaming `Optional::OMITTED` to `Optional::YES` to match the `Optional::NO`. It doesn't seem clear in context what "OMITTED" actually omits.